src/upnp.cpp

```diff
@@ -691,13 +691,13 @@ void upnp::try_map_upnp()
 	TORRENT_ASSERT(is_single_thread());
 	if (m_devices.empty()) return;
 
-	for (auto i = m_devices.begin(), end(m_devices.end()); i != end; ++i)
+	for (auto const& m_device : m_devices)
```

Comment:
this is not a good name for a entry in `m_devices`

Reply:
device better?

Reply:
since the scope is small, I think `d` or `i` would be fine